src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -31,6 +32,16 @@ ov::element::Type get_arithmetic_binary_exec_precision(const std::shared_ptr<ov:
 
     return input_precisions[0];
 }
+
+int float2int(const float value, const ov::element::Type& type = ov::element::f32) {
+    if (type == ov::element::f16) {
```

Comment:
Looks like, a union type could be used here instead of casting.

Reply:
was discussed offline, keep as is